### ISSUE TITLE
JustifiedLayout Columns not redraw correctly

### ISSUE THREAD

**Issue by sbarer** [naver/egjs-infinitegrid#388]
## Description
When using React version, I have set the layoutOptions->column prop to be [0,getNumberOfColumns)], where getNumberOfColumns returns a value between 2 & 5 based on the width of the window. 
I have confirmed that the return value is as expected.
However, when I cross a threshold where the number of columns should change, the JustifiedLayout element does not redraw with the newly specified number of columns, but instead just enlarges/shrinks the photos to fill the bigger/smaller window. 
I have isConstantSize: false, isEqualSize: false as well.
Basically, I want the "none" behaviour of isConstantSize shown in the gif here: https://user-images.githubusercontent.com/3433062/40041387-b98fc9e8-5858-11e8-979a-736126d15310.gif

**Comment by daybrush** [comment]
@sbarer 

Dynamic options are not yet available.

The same and temporary examples as the following issues are provided.

https://github.com/naver/egjs-infinitegrid/issues/371

**Comment by sbarer** [comment]
Thanks for the response! The workaround in the link you provided worked great for me. much appreciated.
